backend/package.json

```diff
@@ -13,6 +13,11 @@
     "nodemon": "^3.0.1",
     "redis": "^4.6.13"
   },
+  "lint-staged": {
+    "**/*.js": [
```

Comment:
Is there any particular reason that this is here and we are not leveraging scripts?

Reply:
what do you mean sir without this can we run or giving more file like ts. @krishnaacharyaa

Reply:
No @Ameerjafar , i mean can't we leverage the existing script "format" instead of npx lint-staged, we can run npm run format in both the places right? So i was trying to understand why we need to have this extra configuration